packages/providers/src/core/PoapTokenApi/PoapTokenApi.ts

```diff
@@ -35,7 +35,7 @@ export class PoapTokenApi implements TokensApiProvider {
    * @param {AuthenticationProvider} [options.authenticationProvider] - Optional provider for JWT authentication.
    */
   constructor({
-    apiKey,
+    apiKey = '',
```

Comment:
Why? POAP API and Compass require API Key. Even if Compass doesn't require it now, it will in a short future.